.github/main.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
```

Comment:
popraw ścieżkę do tego pliku, nie odpaalają Ci się github actions, widać to w PR